### PR TITLE
feat: Add open source history to Experience tab

### LINKED ISSUE #408: List user approved contributions in the opensource section on Experience tab
List the user's approved contributions with the most recent first. in a table format
It should show the fields: project's name, PR description, and date of the contribution.

### PR DESCRIPTION
**Pull request by gabrielcosta42**
Adds open source history to Experience tab

image

Closes #408.

### CODE REVIEW

File: app/admin/user.rb [hunk 1]
```diff
@@ -167,6 +167,18 @@
         end
       end
 
+      tab I18n.t('experience') do
+        panel I18n.t('user_open_source') do
+          table_for user.contributions.valid_pull_requests.order(created_at: :desc), i18n: Contribution, id: 'table_admin_contributions' do
```
Comment (adbatista) [inline app/admin/user.rb:172]: Why the need for assigning a custom id?

File: app/admin/user.rb [hunk 2]
```diff
@@ -167,6 +167,18 @@
...
+            column :name, i18n: Repository do |contribution|
+              contribution.repository.decorate.name
```
Comment (adbatista) [inline app/admin/user.rb:174]: here would be nice to decorate the contribution and, in the decorator add the decorate for the relation

File: db/migrate/20230518170703_add_description_to_contributions.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddDescriptionToContributions < ActiveRecord::Migration[7.0]
+  def change
+    add_column :contributions, :description, :text
```
Comment (adbatista) [inline db/migrate/20230518170703_add_description_to_contributions.rb:3]: almost sure this was added in another PR, merged Friday, can you check?

**Review by adbatista — COMMENTED** [review]
some small comments, just fix the CI and should be ok

File: app/admin/user.rb (hunk 1 above)
Comment (gabrielcosta42) [reply app/admin/user.rb]: Removed this argument, there was no need for it.

File: app/admin/user.rb (hunk 2 above)
Comment (gabrielcosta42) [reply app/admin/user.rb]: Done

File: db/migrate/20230518170703_add_description_to_contributions.rb (hunk 3 above)
Comment (gabrielcosta42) [reply db/migrate/20230518170703_add_description_to_contributions.rb]: It was. Keeping only the older migration now

**Pull request closed without merging**